quickstarts/microsoft.mobilenetwork/mobilenetwork-provision-proxy-sims/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "Provision proxy SIMs",
+    "description": "This template adds the SIM(s) to an existing SIM group and provisions the SIM(s) with an existing mobile network.",
+    "summary": "Adds SIM(s) to an existing SIM group and provision the SIM(s) to be used with an existing mobile network.",
+    "githubUsername": "satravi",
+    "dateUpdated": "2022-06-21",
+    "type": "QuickStart",
+    "environments": [
+        "AzureCloud"
+    ]
+  }
```

Comment:
Add newline to end of file.

Reply:
Not addressed to have the same code style across the templates.